client/my-sites/checkout/composite-checkout/add-items.ts

```diff
@@ -66,7 +66,8 @@ export function createItemToAddToCart( {
 
 	if (
 		( productAlias?.startsWith( 'jetpack_backup' ) ||
-			productAlias?.startsWith( 'jetpack_search' ) ) &&
+			productAlias?.startsWith( 'jetpack_search' ) ||
+			productAlias?.startsWith( 'jetpack_scan' ) ) &&
```

Comment:
Given the changes occurring and the upcoming launch of Anti-Spam, I'd suggest using this pattern established in [another checkout file](https://github.com/Automattic/wp-calypso/pull/43957/files#diff-199e3967ae413c2ff48db966abbf8386R329):
```ts
JETPACK_PRODUCTS_LIST.includes( productAlias )
```

Reply:
Thanks for sharing that!